Psr/Log/Test/LoggerInterfaceTest.php

```diff
@@ -2,16 +2,21 @@
 
 namespace Psr\Log\Test;
 
+if (!class_exists('\\PHPUnit\\Framework\\TestCase', true)) {
```

Comment:
Class names in PHP shouldn't have the leading slash when written in a string. The implementation of class_exist effectively does an ltrim to remove that slash before continuing. The reason why you might see leading slashes is when the ::class notation is used, because then we are resolving names relative to the current namespace.

Reply:
Thanks! fixed. https://github.com/php-fig/log/pull/55/commits/e50f69c5f393e42e21aff1d0c7844184e67c4772